### ISSUE TITLE
Database Seed

### ISSUE THREAD

**Issue by byronferguson** [ethanniser/NextFaster#72]
I'd like to stand this up to play with it, but the included ZIP file of data doesn't really provide all of the necessary information for me to move forward.

What steps do I need to take to spin this up with a populated database?

**Comment by zeluspudding** [comment]
I actually just got this working for myself... and it appears that all of the items are there:

https://github.com/user-attachments/assets/96624f3f-8ecc-445f-b399-8002b077c152

What is it that you're missing, exactly?

**Comment by byronferguson** [comment]
How did you seed the database? I created a a NeonDB, updated the `.env`, and ran the drizzle-kit command outlined in the README. However, the ZIP file is just a SQL dump of the database with COPY from stdin remarks.

I am missing some knowledge of how to populate the database.

**Comment by zeluspudding** [comment]
Yeah... the docs do not explain this. The way I got this to work was to
1. Install Postgres CLI tools (I didn't already have them)
2. Unzip `data.zip` to `data.sql`
3. Then, from that data directory, run
  > psql "YOU_CONNECTION_STRING" -f data.sql

Things were kind of wonky so I had to restart next.js a few times, and finally things to work once I ran `pnpm db:push` again after seeing the DB.

Note that the readme suggests using vercel's postgres service but the free tier version of that cannot actually run the DB (free tier is capped at 250MB and the seeded DB is actually closer to 400MB). Supabase's free tier, however, caps out at 500MB, so I went with that.
